graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -508,10 +518,65 @@ class DgsSchemaProvider(
                         enableInstrumentation
 
                     entityFetcherRegistry.entityFetchers[dgsEntityFetcherAnnotation.name] = dgsComponent to method
+
+                    val type = registry.getType(dgsEntityFetcherAnnotation.name)
+
```

Comment:
nit: Maybe just add a comment that we  are using the config for backwards compatibility and we intend to get rid of it